components/automate-ui/src/app/entities/servers/server.reducer.ts

```diff
@@ -0,0 +1,130 @@
+import { EntityState, EntityAdapter, createEntityAdapter } from '@ngrx/entity';
+import { HttpErrorResponse } from '@angular/common/http';
+import { pipe, set, unset } from 'lodash/fp';
+import { EntityStatus } from 'app/entities/entities';
+import { ServerActionTypes, ServerActions } from './server.actions';
+import { Server } from './server.model';
+
+export interface ServerEntityState extends EntityState<Server> {
+  status: EntityStatus;
+  saveStatus: EntityStatus;
+  saveError: HttpErrorResponse;
+  getStatus: EntityStatus;
+  updateStatus: EntityStatus;
+  deleteStatus: EntityStatus;
+}
+
+// reusable names of the above properties:
+// TODO (tc): Rename this to GET_ALL_STATUS
+const STATUS = 'status';
```

Comment:
Please do what the comment says--rename to `GET_ALL_STATUS` (and `getAllStatus`). Prefer not to propagate a name that is less useful.